app/src/main/java/mozilla/lockbox/view/DialogFragment.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package mozilla.lockbox.view
+
+import android.os.Bundle
+import android.support.v4.app.DialogFragment
+import android.view.View
+import mozilla.lockbox.flux.Presenter
+
+open class DialogFragment : DialogFragment() {
```

Comment:
Hmm. I see this makes sense.

I should file a follow up to rename `CommonFragment` to `mozilla.lockbox.view.Fragment`.